kart/tabular/working_copy/postgis.py

```diff
@@ -337,3 +343,14 @@ def _apply_meta_schema_json(self, sess, dataset, src_value, dest_value):
             sess.execute(
                 f"""ALTER TABLE {self.table_identifier(table)} ALTER COLUMN {self.quote(col.name)} TYPE {dest_type};"""
             )
+
+    def _check_for_unsupported_ds_types(self, sess, target_datasets, schema="public"):
+        for target_dataset in target_datasets:
```

Comment:
target_datasets is a dict like { "dataset-path": dataset}
If you just want the dataset part of it, just iterate over it like this:
`for dataset in target_datasets.values()`
Or if you decide you need the path too, then do this:
`for dataset_path, dataset in target_datasets.items()`